src/main/java/teammates/ui/request/FeedbackSessionUpdateRequest.java

```diff
@@ -1,8 +1,54 @@
 package teammates.ui.request;
 
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * The update request of a feedback session.
  */
 public class FeedbackSessionUpdateRequest extends FeedbackSessionBasicRequest {
 
+    private Map<String, Long> studentDeadlines;
+    private Map<String, Long> instructorDeadlines;
+
+    private boolean isGoingToNotifyAboutDeadlines;
+
+    /**
+     * Gets the deadlines for students.
+     */
+    public Map<String, Instant> getStudentDeadlines() {
+        Map<String, Instant> studentDeadlineInstants = new HashMap<>();
+        studentDeadlines.forEach((emailAddress, deadline) -> {
+            studentDeadlineInstants.put(emailAddress, Instant.ofEpochMilli(deadline));
+        });
```

Comment:
Would this be cleaner?

```suggestion
        return studentDeadlines.entrySet().stream()
                .collect(Collectors.toMap(Map.Entry::getKey, e -> Instant.ofEpochMilli(e.getValue())));
```